lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+	attr_accessor :cells, :index
+
+	def initialize
+	    @cells = []
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.